### DISCUSSION TITLE
New Payment Gateway

### DISCUSSION THREAD

**Discussion by Ansh1177** [Paymenter/Paymenter#397] [Ideas]
Hi, Can you please add Razorpay because it is one of the most popular payment gateway and used by many people.

**Comment by GomorrhaDev** [comment]
There is currently one person from the community who is working on a Razorpay addon.

**Comment by sarthakjainxyz** [comment]
yeah its finished now you can download it from here: https://market.paymenter.org/extensions/36/RazorPay

**Reply by GomorrhaDev** [reply]
@Ansh1177

**Reply by Ansh1177** [reply]
Thanks Guys it really means alot.
